src/components/Photo/DocumentLiveCapture.js

```diff
@@ -86,13 +86,15 @@ export default class DocumentLiveCapture extends Component<Props, State> {
     const documentSize = id1SizeDocuments.has(documentType)
       ? 'id1Card'
       : 'id3Card'
+    const idealCameraHeightInPixels = 1080
```

Comment:
I'm quite unconfident with literal constant declarations like this. We have a `cameraHeight` constant declared in `Camera/index.js`, I'm thinking about grouping them in a single place and reuse when possible.

```suggestion
    const IDEAL_CAMERA_HEIGHT_IN_PIXELS = 1080
    const FALLBACK_CAMERA_HEIGHT_IN_PIXELS = 720
```